py_gtfs_rt_ingestion/tests/test_batcher.py

```diff
@@ -48,16 +49,14 @@ def test_batch_class(capfd) -> None:  # type: ignore
     for filename, filesize in files.items():
         batch.add_file(filename=filename, filesize=filesize)
     # Checking Batch __str__ method
-    print(batch)
-    out, _ = capfd.readouterr()
+    out = str(batch)
     assert (
         out == f"Batch of {sum(files.values())} bytes in {len(files)} "
-        f"{config_type} files\n"
+        f"{config_type} files"
     )
 
-    # Calling `trigger_lambda` method raises exception
-    with pytest.raises(ArgumentException):
-        Batch(ConfigType.RT_VEHICLE_POSITIONS).trigger_lambda()
+    Batch(ConfigType.RT_VEHICLE_POSITIONS).trigger_lambda()
+    assert "failed=trigger_lambda" in caplog.text
```

Comment:
this was failing since this method no longer raises. check that it does log a failure.